File: frontend/src/types/entreprise.ts
export interface Entreprise {
  id: string;
  nom: string;
  ice: string | null;
  identifiant_fiscal: string | null;
  rc: string | null;
  is_active?: boolean;
  creee_automatiquement: boolean;
  ecritures_brouillon?: number;
  ecritures_a_verifier?: number;
  ecritures_validees?: number;
}

File: backend/app/schemas/entreprise.py
"""
app/schemas/entreprise.py

Schéma de sortie pour lister les entreprises d'un cabinet — alimente la
colonne "Entreprises" de la page Chronos.
"""
import uuid
from pydantic import BaseModel, ConfigDict


class EntrepriseOut(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: uuid.UUID
    nom: str
    ice: str | None
    creee_automatiquement: bool


class EntrepriseDisponibleOut(EntrepriseOut):
    ecritures_brouillon: int = 0
    ecritures_a_verifier: int = 0
    ecritures_validees: int = 0
